pkg/config/piped.go

```diff
@@ -919,13 +919,18 @@ func (n *NotificationReceiver) Mask() {
 }
 
 type NotificationReceiverSlack struct {
-	HookURL string `json:"hookURL"`
+	HookURL    string `json:"hookURL"`
+	OAuthToken string `json:"oauthToken"`
+	ChannelID  string `json:"channelID"`
```

Comment:
Could you add validation for this struct? I think we need something like
- Ensure that if HookURL already available, no OAuthToken / ChannelID be set (since we only use one of them), and the revert condition as well.
- If the OAuthToken is available ChannelID is required as well since we need that to push message in case of send message via API